.gitmodules

```
@@ -256,3 +256,7 @@
 [submodule "contrib/sysroot"]
 	path = contrib/sysroot
 	url = https://github.com/ClickHouse-Extras/sysroot.git
+[submodule "contrib/oneTBB"]
+	path = contrib/oneTBB
+	url = https://github.com/zhanglistar/oneTBB.git
```

Why not original repo?
There is commit that you are referenced (4e021eafc03519cb1081ed775c9502cae72f8d8d)

```suggestion
	url = https://github.com/oneapi-src/oneTBB.git
```